livekit-plugins/livekit-plugins-rime/livekit/plugins/rime/tts.py

```diff
@@ -133,6 +138,8 @@ def __init__(
         self._session = http_session
         self._base_url = base_url
 
+        self._tts_http_timeout = ARCANA_MODEL_TIMEOUT if model == "arcana" else MISTV2_MODEL_TIMEOUT
```

Comment:
nit: let's call this `total_timeout` ? to match aiohttp parameter

tts feels redundant in context of the TTS class

Reply:
yea good point.

Reply:
changed